README.md

```diff
@@ -128,6 +128,12 @@ You can alter default [Flask](http://flask.pocoo.org/docs/1.0/config/) or
     # Default product to display (picks first available)
     CUBEDASH_DEFAULT_PRODUCTS = ('ls8_nbar_albers', 'ls7_nbar_albers')
     
+    # CUBEDASH_PRODUCT_GROUP_BY_REGEX is a list of tuples of (regex_pattern_string,group_name)
+    # If re.search(regex, product name) then return the corresponding group.
+    # If CUBEDASH_PRODUCT_GROUP_BY_REGEX string parses then regex matching will be used in place of CUBEDASH_PRODUCT_GROUP_BY_FIELD.
+    # If CUBEDASH_PRODUCT_GROUP_BY_REGEX string is malformed an error will be logged and we default back to CUBEDASH_PRODUCT_GROUP_BY_FIELD.
+    # eg "[('.*albers.*','Albers projection'), ('.*level1.*','Level 1 products'), ('.*usgs.*','Level 1 products')]" 
+    CUBEDASH_PRODUCT_GROUP_BY_REGEX = None 
     # Which field should we use when grouping products in the top menu?
     CUBEDASH_PRODUCT_GROUP_BY_FIELD = 'product_type'
```

Comment:
More consistent language with the other options on the page:

```suggestion
    # Specify product grouping in the top menu.
    # Expects a series of `(regex, group_label)` pairs. Each product will be grouped into the first regexp that matches
    # anywhere in its name. Unmatched products have their own group.
    # eg "(('^usgs_','USGS products'), ('_albers$','C2 Albers products'), ('level1','Level 1 products'), )" 
    CUBEDASH_PRODUCT_GROUP_BY_REGEX = None
    # Otherwise, group by a single metadata field in the products:
    CUBEDASH_PRODUCT_GROUP_BY_FIELD = 'product_type'
```

(Also removed the error handling explanation as suggested below)